lib/active_merchant/billing/gateways/braintree_blue.rb

```diff
@@ -589,6 +589,10 @@ def create_transaction_parameters(money, credit_card_or_vault_id, options)
           parameters[:recurring] = true
         end
 
+        if options[:transaction_source]
+          parameters[:transaction_source] = options[:transaction_source]
+        end
+
```

Comment:
It looks like we are letting customers drive setting the `recurring` and `transaction_source` fields. Is there any concern that we may set `recurring` to `true` and the `transaction_source` would not be `recurring`? Does one field take priority over the other?

Reply:
I could also see an argument for not limiting this, and leaving it to the merchant to use the fields the right way. I don't _think_ this hurts, though.